routers/apiMethods.js

```diff
@@ -0,0 +1,116 @@
+const {returnJson} = require('./../async-util');
+const apiConstants = require('./../chain/conf/api_constants.json');
+
+const {createSwap} = require('./../service');
+const {findSwapOutpoint} = require('./../service');
+const {getAddressDetails} = require('./../service');
+const {getExchangeRates} = require('./../service');
+const {getInvoiceDetails} = require('./../service');
+const {checkSwapStatus} = require('./../service');
+const {broadcastTransaction} = require('./../service');
+
+const reqGetAddressDetails = function (args) {
+  return getAddressDetails({
+      address: args.req.params.address,
+      network: args.req.params.network
+    }, returnJson({log: args.log, res: args.res})
+  );
+};
+
+const reqGetExchangeRates = function (args) {
```

Comment:
Style would be to destructure here: `({cache, log, res}) => {}` - args make it harder to see what is being passed. But also functions should have parameters detailed above so you know what you are passing